### DISCUSSION TITLE
ESP32 USB CDC

### DISCUSSION THREAD

**Discussion by Saeedsaeb** [adafruit/Adafruit_TinyUSB_Arduino#197] [Q&A]
Hello ,
I am trying to make a USB device (caller ID) which is powered by an STM32 Device in CDC Serial ,into a WIFI device ,
the program is quite simple since its just a bridge .

#error "Currently multiple CDCs on ESP32-Sx is not yet supported. An PR to update core/esp32/USBCDC and/or pre-built libusb are needed."
// for ESP32, we need to specify instance number when declaring object

I wanted to ask if this problem has been solved , I am getting the error obviously , or if I can do something like getting libusb library from somewhere , thank you

**Comment by hathach** [comment]
not yet, when we resolve this, we will remove this line.
